node/api/hostdb.go

```diff
@@ -35,8 +35,33 @@ type (
 		Entry          ExtendedHostDBEntry        `json:"entry"`
 		ScoreBreakdown modules.HostScoreBreakdown `json:"scorebreakdown"`
 	}
+
+	// HostDBGet holds information about the hostdb.
+	HostDBGet struct {
+		InitialScanComplete bool                  `json:"initialscancomplete"`
+		QueuedScans         []modules.HostDBEntry `json:"queuedscans"`
```

Comment:
Returning a large list of hostdb entries might be too much data to pass through the API, especially if it's early and there are thousands of scans remaining.

Reply:
Yeah I wasn't sure about that. It introduces a lot of overhead but I'm wondering if it would be useful to have an endpoint for the UI that lets you check information about hosts that are currently being scanned.